modules/enabling_encapsulation.adoc

```diff
@@ -0,0 +1,157 @@
++
+[source,yaml]
+----
+  apiVersion: machineconfiguration.openshift.io/v1
+  kind: MachineConfig
+  metadata:
+    labels:
+      machineconfiguration.openshift.io/role: master
+    name: 99-kubens-master
+  spec:
+    config:
+      ignition:
+        version: 3.2.0
+      systemd:
+        units:
+        - enabled: true <1>
+          name: kubens.service
+  ---
+  apiVersion: machineconfiguration.openshift.io/v1
+  kind: MachineConfig
+  metadata:
+    labels:
+      machineconfiguration.openshift.io/role: worker
+    name: 99-kubens-worker
+  spec:
+    config:
+      ignition:
+        version: 3.2.0
+      systemd:
+        units:
+        - enabled: true <2>
+          name: kubens.service
+----
+<1> If `enabled` is set to true, encapsulation is in effect for all control plane (also known as "master") nodes.
+<2> If `enabled` is set to true, encapsulation is in effect for all worker nodes.
+
+. Apply the mount namespace `MachineConfig` CR:
++
+[source,terminal]
+----
+$ oc apply -f mount_namespace_config.yaml
+----
++
+.Example output
+[source,terminal]
+----
+machineconfig.machineconfiguration.openshift.io/99-kubens-master created
+machineconfig.machineconfiguration.openshift.io/99-kubens-worker created
+----
+
+. The `MachineConfig` CR can take up to 30 minutes to finish being applied in the cluster. You can check the status of the `MachineConfig` CR by running the following command:
++
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Example output
+[source,terminal]
+----
+NAME     CONFIG                                             UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
+master   rendered-master-03d4bc4befb0f4ed3566a2c8f7636751   False     True       False      3              0                   0                     0                      45m
+worker   rendered-worker-10577f6ab0117ed1825f8af2ac687ddf   False     True       False      3              1                   1
+----
+
+. Wait for the `MachineConfig` CR to be applied successfully across all control plane and worker nodes. This can be done as follows:
++
+[source,terminal]
+----
+$ oc wait --for=condition=Updated mcp --all --timeout=30m
+----
++
+.Example output
+[source,terminal]
+----
+machineconfigpool.machineconfiguration.openshift.io/master condition met
+machineconfigpool.machineconfiguration.openshift.io/worker condition met
+----
+
+.Verification
+
+To establish whether or not encapsulation is enabled, run the following commands:
+
+. Open a debug shell to the cluster host and open a `chroot` session:
++
+[source,terminal]
+----
+$ oc debug node/<node_name>
+----
++
+[source,terminal]
+----
+sh-4.4# chroot /host
+----
+
+. Check systemd mount namespace:
++
+[source,terminal]
+----
+sh-4.4# readlink /proc/1/ns/mnt
+----
++
+.Example output
+[source,terminal]
+----
+mnt:[4026531953]
+----
+
+. Check Kubelet mount namespace:
```

Comment:
Kubelet is the name of the software, and kubelet is the name of the executable.  We probably want Kubelet most places in this document except in code blocks.

Reply:
Done.

Reply:
I had to check becuase I was wondering about this in another PR: https://redhat-documentation.github.io/supplementary-style-guide/#kubelet